src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```
@@ -221,7 +224,7 @@ protected void shutdownService() {
       if(selector != null && selector.isOpen()) {
         selector.close();
       }
-    } catch (IOException e) {
+    } catch (Exception e) {
```

What are you swallowing just any exception here?